src/main/kotlin/com/terraformation/backend/accelerator/ParticipantProjectSpeciesService.kt

```diff
@@ -81,6 +83,33 @@ class ParticipantProjectSpeciesService(
     }
   }
 
+  /*
+   * When a species is updated, if it belongs to an organization with participants and is associated
+   * to a participant project, we need to update its status to "in review" across all
+   * associated projects. This only applies to users with no accelerator related global roles.
+   */
+  @EventListener
+  fun on(event: SpeciesEditedEvent) {
+    if (currentUser().canReadAllAcceleratorDetails()) {
+      return
+    }
```

Comment:
How will this work when doing manual testing of the feature? I'm imagining someone with admin privileges on staging trying this out by editing a species in their own test organization using the non-admin UI; this permission check would mean the status of the submission wouldn't change.

Maybe it's okay to tell people to log in with separate non-admin accounts for testing this, though.

Reply:
I will make sure to mention this in the testing notes, but yes, they will need to log in to a non-admin account to see the status change.